examples/spice.mzLib.txt

```diff
@@ -0,0 +1,646 @@
+<mzSpecLib 1.0>
+MS:1003188|library name=spice
+MS:1003190|library version=1.0
+MS:1001017|release date=7/7/2023 7:06:42 PM
+<AttributeSet Spectrum=all>
+MS:1000008|ionization type=MS:1000073|electrosprary ionization
+MS:1000465|scan polarity=MS:1000130|positive scan
+<Spectrum=1>
+MS:1003203|constituent spectrum file=mzspec:USI000000:spice_ph/cal_10ng.raw:scan:795

```

Comment:
For a singleton spectrum identified by USI, the attribute to use is `MS:1003063|universal spectrum identifier`
This is equivalent to:
```
MS:1003203|constituent spectrum file=mzspec:USI000000:spice_ph/cal_10ng.raw
MS:1003057|scan number=795
```

Reply:
fixed.